pkg/http/core/search.go

```diff
@@ -1,11 +1,133 @@
 package core
 
 import (
+	"errors"
+	"fmt"
 	"net/http"
+	"strconv"
+	"strings"
 
+	clouddriver "github.com/billiford/go-clouddriver/pkg"
+	"github.com/billiford/go-clouddriver/pkg/sql"
 	"github.com/gin-gonic/gin"
 )
 
+type SearchResponse []Page
+
+type Page struct {
+	PageNumber   int          `json:"pageNumber"`
+	PageSize     int          `json:"pageSize"`
+	Query        string       `json:"query"`
+	Results      []PageResult `json:"results"`
+	TotalMatches int          `json:"totalMatches"`
+	// Platform     string       `json:"platform"`
```

Comment:
Please remove this comment if not needed

Reply:
We've been leaving unused struct fields commented in this project for future use.

This one in particular is useless - but I think we should leave it to align with the rest of the project.